File: src/Bpo.Dms.WebApp/Models/DataLoadRequest.cs
namespace Bpo.Dms.WebApp.Models
{
    public class DataLoadRequest
    {
        public int PageSize { get; set; }
        public int CurrentPage { get; set; }
        public string SearchTerm { get; set; }
        public IEnumerable<SortInfo> Sorts { get; set; }
        public class SortInfo
        {
            public string Name { get; set; }
            public bool IsDesc { get; set; }
        }
    }
}

File: src/Bpo.Dms.WebApp/ClientApp/src/services/models/DataLoadRequest.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { SortInfo } from './SortInfo';

export type DataLoadRequest = {
    pageSize?: number;
    currentPage?: number;
    searchTerm?: string | null;
    sorts?: Array<SortInfo> | null;
};